aips/data_loaders/index_time_boosts.py

```diff
@@ -0,0 +1,31 @@
+from engines.opensearch.OpenSearchCollection import OpenSearchCollection
+from engines.solr.SolrCollection import SolrCollection
+from pyspark.sql.functions import collect_list, create_map
+from aips.spark.dataframe import from_sql
+from aips.spark import create_view_from_collection
+
+def load_dataframe(boosted_products_collection, boosts_collection):
+    create_view_from_collection(boosts_collection,
+                                boosts_collection.name)    
+    create_view_from_collection(boosted_products_collection,
+                                boosted_products_collection.name)    
+    assert(type(boosted_products_collection) == type(boosts_collection))
+    if isinstance(boosted_products_collection, SolrCollection):
```

Comment:
Let's add a "get_engine_name" function or similar to the Collection class. This removes the need to import all the Collections to do `isinstance` checks in various places throughout the code base.

Reply:
`get_engine_name` has been implemented